### PR TITLE
Update pt_br.po

### PR DESCRIPTION
**Pull request by porres**

### CODE REVIEW

**Comment by umlaeute** [comment]
the problem why this PR keeps failing is, that the `.po`-file format requires ordinary double-quotes `"`, whereas your patch introduces left-right styled quotes (`“` and `”`).

i think this is a problem with the editor you were using to edit the file.

File: po/pt_br.po [hunk 1]
```diff
@@ -671,11 +671,11 @@ msgstr "2. Copiar o conteúdo para %s."
 
 #, tcl-format
 msgid "[deken] uninstalling '%s'"
-msgstr ""
+msgstr “[deken] desinstalando '%s'”
```
Comment (umlaeute) [inline po/pt_br.po]: check the quotes

**Comment by umlaeute** [comment]
as i seem to be unable to directly push to this PR, here's a diff (apply with `git am 0001-Fixed-syntax-errors-in-quotes.patch`)

~~~patch
From cc749019d6692ebc515b3922faa027b55a51b457 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?IOhannes=20m=20zm=C3=B6lnig?= <zmoelnig@iem.at>
Date: Thu, 13 Sep 2018 12:05:17 +0200
Subject: [PATCH] Fixed syntax errors in quotes

---
 po/pt_br.po | 21 +++++++++++----------
 1 file changed, 11 insertions(+), 10 deletions(-)

diff --git a/po/pt_br.po b/po/pt_br.po
index 53b417b7..76fcfcda 100644
--- a/po/pt_br.po
+++ b/po/pt_br.po
@@ -671,11 +671,11 @@ msgstr "2. Copiar o conteúdo para %s."
 
 #, tcl-format
 msgid "[deken] uninstalling '%s'"
-msgstr “[deken] desinstalando '%s'”
+msgstr "[deken] desinstalando '%s'"
 
 #, tcl-format
 msgid "Uninstalling %1$s from %2$s failed!"
-msgstr “Falha ao desinstalar %1$s de %2$s!"
+msgstr "Falha ao desinstalar %1$s de %2$s!"
 
 #, tcl-format
 msgid "[deken] deken-plugin.tcl (Pd externals search) loaded from %s."
@@ -717,22 +717,23 @@ msgid "Try to remove libraries before (re)installing them?"
 msgstr "Tentar remover bibliotecas antes de reinstala-las?"
 
 msgid "Show README of newly installed libraries (if present)?"
-msgstr “Mostrar README de bibliotecas recém instaladas (se presente)?”
+msgstr "Mostrar README de bibliotecas recém instaladas (se presente)?"
 
 msgid "Should newly installed libraries be added to Pd's search path?"
-msgstr “Bibliotecas recém instaladas devem ser adicionadas aos caminhos de busca do Pd?"
+msgstr "Bibliotecas recém instaladas devem ser adicionadas aos caminhos de busca do Pd?"
 
 msgid "Platform settings:"
-msgstr “Configurações de platforma"
+msgstr "Configurações de platforma"
 
 #, tcl-format
 msgid "Default platform: %s"
-msgstr “Plataforma padrão: %s"
+msgstr "Plataforma padrão: %s"
 
 msgid "User-defined platform:"
-msgstr "platforma definida pelo usuário“
+msgstr "platforma definida pelo usuário"
+
 msgid "Hide foreign architectures?"
-msgstr “Esconder arquiteturas estrangeiras?"
+msgstr "Esconder arquiteturas estrangeiras?"
 
 #, tcl-format
 msgid "Deken %s Preferences"
@@ -817,7 +818,7 @@ msgid "Install package"
 msgstr "Instalar pacote"
 
 msgid "Open webpage"
-msgstr “Abrir página da web”
+msgstr "Abrir página da web"
 
 msgid "_"
 msgstr ""
@@ -874,7 +875,7 @@ msgid "Print..."
 msgstr "Imprimir..."
 
 msgid "Paste Replace"
-msgstr “Colar Substituir”
+msgstr "Colar Substituir"
 
 msgid "Duplicate"
 msgstr "Duplicar"
-- 
2.18.0

~~~

**Comment by danomatika** [comment]
@porres If you were using TextEdit, I *highly* recommend turning off "Smart quotes" in the preferences. Better yet, use [Poedit](https://poedit.net).

**Comment by danomatika** [comment]
Thanks @umlaeute. This is in now, @porres.

**Comment by porres** [comment]
> use Poedit.

ok, thanks

**Pull request merged**